css-scroll-snap-2/Overview.bs

```diff
@@ -275,17 +275,125 @@ Snap Events {#snap-events}
 	<table class="data" id="eventhandlers">
 		<thead>
 			<tr>
-				<th>Event handler
-				<th>Event handler event type
+				<th>Event</th>
+				<th>Interface</th>
+				<th>Targets</th>
+				<th>Description</th>
+			</tr>
+		</thead>
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanged" event>snapchanged</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>scroll containers</td>
+				<td>Fired at the scrolling element or {{Document}} at the end of a scroll (before a {{scrollend}} event)
+					or after a <a href="https://drafts.csswg.org/css-scroll-snap-1/#re-snap">layout snap</a>
+					if the element that the scrolling element or Document is snapped to changed.</td>
+			</tr>
 			<tr>
-				<th><dfn event>snapChanging</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanging" event>snapchanging</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>snap containers</td>
+				<td>Fired at the scrolling element or {{Document}} during scrolling (similar to {{scroll}} events),
+					(before a {{snapchanged}} event, if one fires) if the scrolling will lead to a change in the
+					element that the scrolling element or Document is snapped to.</td>
+			</tr>
+		</tbody>
 	</table>
 
+	<h4 for="snapchanged" id="snapchanged"> snapchanged </h4>
+	{{snapchanged}} indicates that the snap area to which a snap container is snapped along either axis has changed.
+	{{snapchanged}} should be dispatched:
+
+	<ol>
+	<li>
+		at the end of a scrolling operation, after the user agent has completed its
```

Comment:
This could probably refer to the completed term in cssom-view, https://drafts.csswg.org/cssom-view/#scroll-completed . Note that we should also specify the relative order with respect to scroll and scrollend events, which is further reason to make this part of cssom-view as suggested by @zcorpan here: https://github.com/w3c/csswg-drafts/pull/9515#discussion_r1403693509

We could for now add it as an appendix, similar to scroll-animation additions to the other animations specs: https://www.w3.org/TR/scroll-animations-1/#timeline-name-scope

Reply:
I've specified in the snapchanged and snapchanging sections that snapchanging events should be fired before scroll events and that snapchanged events should be fired before scrollend events. I've also added Appendix which describes an algorithm where snapchanging should be fired during scrolling and snapchanged should be fired after scrolling is completed and referenced scroll-completed from cssom-view.